command/csi_volume.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type CSIVolumeCommand struct {
+	Meta
+}
+
+func (c *CSIVolumeCommand) Help() string {
+	helpText := `
+Usage: nomad csi volume <subcommand> [options]
+
+  csi volume groups all the CSI (Container Storage Interface) commands that
+  operate on volumes. CSI Volumes must be created in the remote provider before
+  the volume is available to a nomad task.
+
+  Register a new volume or update an existing volume:
+
+      $ nomad csi volume register <input>
+
+  Examine the status of a volume:
+
+      $ nomad csi volume status <id>
+
+  List existing volumes:
+
+      $ nomad csi volume list
+
+  Deregister a volume, removing it from the system:
```

Comment:
"Removing it from the system" sounds like it does a lot more than it does. Maybe just cut it down to "Deregister an unused volume"?